index 1.js

```diff
@@ -0,0 +1,19 @@
+"use strict"
+
+function findPrimeNumbers(input) {
+    const primeNumbers = [];
+    for (let i = 2; i <= input; i++) {
+        let isPrime = true;
+        for (let j = 2; j < i; j++) {
+            if (i % j === 0) {
+                isPrime = false;
```

Comment:
Is it necessary to use the `isPrime` variable?

Reply:
In this particular approach, I think yes it's necessary.